crates/bevy_asset/src/lib.rs

```diff
@@ -101,13 +104,26 @@ pub enum AssetMode {
     Processed,
 }
 
+/// Configures how asset meta will be used.
+#[derive(Debug, Default, Clone)]
+pub enum AssetMetaMode {
+    /// Always assume assets have meta files.
+    #[default]
+    Always,
```

Comment:
this should then crash if the meta file is missing instead of defaulting

Reply:
Fair point. Imo crashing isn't the desired behavior here. The "opt in meta file" pattern is very nice. I'll adjust the name and docs to indicate that it will always _check_.